module/tools/SystemConfiguration/data/config/nugus2/SystemConfiguration.yaml

```diff
@@ -0,0 +1,9 @@
+# Systemd units that should be enabled
```

Comment:
These are identical!

Reply:
They are, but we have to allow for the wifi interface to be named differently, and the way config merging works forces us to replicate the entire list

Reply:
Yeah but replicate it when it's different, it's the same for every single robot at the moment

Reply:
Because I dont know the wifi interface name for the robots lol